pkg/provider/azure_loadbalancer.go

```diff
@@ -328,46 +328,7 @@ func (az *Cloud) getLoadBalancerResourceGroup() string {
 // according to the mode annotation on the service. This could be happened when the LB selection mode of an
 // existing service is changed to another VMSS/VMAS.
 func (az *Cloud) shouldChangeLoadBalancer(service *v1.Service, currLBName, clusterName string) bool {
```

Comment:
do we still need to keep this function?

Reply:
yes we do

Reply:
why? for future use?

Reply:
Yes, the logic will be similar in the new design of multi slb.

Reply:
The basic LB still needs this function. This one should not be deleted.